modules/nw-egress-ips-static.adoc

```diff
@@ -44,6 +44,11 @@ $ oc patch netnamespace project1 --type=merge \
 +
 To provide high availability, set the `egressIPs` value to two or more IP addresses on different nodes. If multiple egress IP addresses are set, then pods use all egress IP addresses roughly equally.
 
```

Comment:
Replace this blank line with a `+` to get the Note to hang off the step

Reply:
Done. Thanks! All rendering just fine.